docs/user/dashboard/dashboard.asciidoc

```diff
@@ -333,69 +333,75 @@ Copy panels from one dashboard to another dashboard.
 . On the *Copy to dashboard* window, select the dashboard, then click *Copy and go to dashboard*.
 
 [float]
-[[search-or-filter-your-data]]
-== Search and filter your data
-
-{kib} supports several ways to search your data and apply {es} filters. You can combine the filters with any panel
-filter to display the data want to you see.
+[[add-dashboard-settings]]
+== Add the dashboard settings
 
-For more information about {kib} and {es} filters, refer to <<kibana-concepts-analysts,{kib} concepts>>.
+Add the title, tags, design options, and more to the dashboard.
 
-To apply a panel-level time filter:
+. In the toolbar, click *Settings*.
 
-. Open the panel menu, then select *More > Edit panel settings*.
+. On the *Dashboard settings* flyout, enter the *Title* and an optional *Description*.
 
-. Select *Apply a custom time range*.
+. Add any applicable <<managing-tags,*Tags*>>. 
 
-. Enter the time range you want to view, then click *Save*.
+. Specify the following settings:
 
-To view and edit panel-level filters:
+* *Store time with dashboard* &mdash; Saves the specified time filter.
 
-. On the panel, click image:images/dashboard_panelFiltersButton_8.7.0.png[Panel filters button on panel header].
-
-. To edit, click *Edit filters*.
+* *Use margins between panels* &mdash; Adds a margin of space between each panel.
 
-[float]
-[[apply-design-options]]
-== Apply design options
+* *Show panel titles* &mdash; Displays the titles in the panel headers.
 
-Apply a set of design options to the entire dashboard.
+* *Sync color pallettes across panels* &mdash; Applies the same color palette to all panels on the dashboard.
 
-. If you're in view mode, click *Edit* in the toolbar.
+* *Sync cursor across panels* &mdash; When you hover your cursor over a *Lens*, *TSVB*, aggregation-based, or *Timelion* XY chart and a heatmap chart, the cursor on all other related dashboard charts automatically appear.
```

Comment:
**Nit:** The phrasing is a bit awkward here IMO... A couple suggestions, but take 'em or leave 'em haha :)

```suggestion
* *Sync cursor across panels* &mdash; When you hover your cursor over a *Lens*, *TSVB*, aggregation-based, or *Timelion* XY chart and a heatmap chart, the cursor on all other related dashboard charts automatically appears.
```

Or

```suggestion
* *Sync cursor across panels* &mdash; When you hover your cursor over a *Lens*, *TSVB*, aggregation-based, or *Timelion* XY chart and a heatmap chart, the cursor on all other related dashboard charts will automatically appear.
```